ThunderCloud/QuizQuestionContainerViewController.swift

```diff
@@ -283,7 +355,20 @@ open class QuizQuestionContainerViewController: UIViewController {
         popToLastViewController(excluding: [
             questionContainerClass ?? QuizQuestionContainerViewController.self,
             quizCompletionClass
-            ])
+        ])
+    }
+    
+    open override func accessibilitySettingsDidChange() {
+        
+        navigationController?.navigationBar.barTintColor = ThemeManager.shared.theme.navigationBarBackgroundColor
+        navigationController?.navigationBar.tintColor = ThemeManager.shared.theme.navigationBarTintColor
+        
+        redraw()
+        redrawContinueButton()
```

Comment:
Does `redrawSelectedLabel` also need to be called?

Reply:
It's handled in the two lines below 😊 There's no need to call the whole function, because we don't need to update the text on it!